doc/source/cluster/running-applications/job-submission/openapi.yml

```diff
@@ -0,0 +1,471 @@
+openapi: 3.1.0
+info:
+  title: Ray Jobs API
+  version: 4.0.0
+  description: |
+    This is the specification for the Ray Jobs REST API.
+    See the Ray Jobs documentation for details:
+    https://docs.ray.io/en/latest/cluster/running-applications/job-submission/index.html
+
+paths:
+  /api/version:
+    get:
+      summary: |
+        Get Version
+      description: |
+        Get the Ray Jobs API version and the Ray version running on the cluster.
+      responses:
+        200:
+          summary: Version
+          description: |
+            The Ray Jobs API version and the Ray version running on the cluster.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  version:
+                    description: The version of the Ray Jobs API running on the server.
```

Comment:
Which one reads declaratively:
"The version of the Ray Jobs API running on the server." or
"Ray Jobs' API version running on the server."